arch/src/aarch64/gicv2.rs

```diff
@@ -0,0 +1,97 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::{boxed::Box, result};
+
+use kvm_ioctls::{DeviceFd, VmFd};
+
+use super::gic::{finalize_device, init_device, set_device_attribute, Error, GICDevice};
+
+type Result<T> = result::Result<T, Error>;
+
+// Unfortunately bindgen omits defines that are based on other defines.
+// See arch/arm64/include/uapi/asm/kvm.h file from the linux kernel.
+const KVM_VGIC_V2_DIST_SIZE: u64 = 0x1000;
+const KVM_VGIC_V2_CPU_SIZE: u64 = 0x2000;
```

Comment:
Please move this constants in the `impl GICv2 {` block since they are GicV2 related. Please try to define all the methods / constants where they belong.